### ISSUE TITLE
Hyperparameters

### ISSUE THREAD

**Issue by dylanrandle** [capstone2019-neuralsearch/AC297r_2019_NAS#17]
Figure out which parameters to tune to make DARTS work (base dataset: MNIST/FashionMNIST)

**Comment by dylanrandle** [comment]
It appears that simply changing the learning rate from 0.025 to a more sensible 0.001 fixes the problem. 

My conjecture is that with such a high learning rate, the model was jumping around in parameter space, always ending in local minima that do well on the training set, but only sometimes ones that generalize and perform well on validation.
